### ISSUE TITLE
hubble-relay chart does not have PodSecurityPolicy

### ISSUE THREAD

**Issue by ialidzhikov** [gardener/gardener-extension-networking-cilium#90]
/area networking
/kind bug

**What happened**:
Cannot enable successfully hubble when Shoot `.spec.kubernetes.allowPrivilegedContainers=false`.

**What you expected to happen**:
To be able to enable hubble when Shoot `.spec.kubernetes.allowPrivilegedContainers=false`.

**How to reproduce it (as minimally and precisely as possible)**:
1. Create a Shoot with `.spec.kubernetes.allowPrivilegedContainers=false` and the following `providerConfig`:
```yaml
    providerConfig:
      apiVersion: cilium.networking.extensions.gardener.cloud/v1alpha1
      kind: NetworkConfig
      hubble:
        enabled: true
```

2. Make sure that hubble-relay Pod cannot be created with reason:
```
$ k -n kube-system describe rs hubble-relay-5f58b96d47

Events:
  Type     Reason        Age                  From                   Message
  ----     ------        ----                 ----                   -------
  Warning  FailedCreate  17m (x9 over 17m)    replicaset-controller  Error creating: pods "hubble-relay-5f58b96d47-" is forbidden: PodSecurityPolicy: unable to admit pod: []
  Warning  FailedCreate  6m50s (x9 over 17m)  replicaset-controller  Error creating: pods "hubble-relay-5f58b96d47-" is forbidden: PodSecurityPolicy: unable to admit pod: [spec.volumes[0]: Invalid value: "hostPath": hostPath volumes are not allowed to be used]
```

**Environment**:

- Gardener version (if relevant):
- Extension version: v1.11.0
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Others:

**Comment by jguipi** [comment]
any update on this issue ?

**Comment by ScheererJ** [comment]
This was first addressed with #130. Now #143 addresses the remaining issue with the seccomp profiles.
